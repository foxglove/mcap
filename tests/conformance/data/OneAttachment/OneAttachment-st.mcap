version https://git-lfs.github.com/spec/v1
oid sha256:f1cf96f2037f2061457c754a00c0d0550e6f923e8e2443a7fe351b152d540ea6
size 208
