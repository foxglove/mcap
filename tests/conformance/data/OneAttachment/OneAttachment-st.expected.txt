Header library=s:"" profile=s:""
Attachment content_type=s:application/octet-stream created_at=n:1 data=b:<010203> log_time=n:2 name=s:myFile
Statistics attachment_count=i:1 channel_count=i:0 channel_message_counts=m:{} chunk_count=i:0 message_count=n:0
AttachmentIndex content_type=s:application/octet-stream data_size=n:3 length=n:78 log_time=n:2 name=s:myFile offset=n:25
Footer summary_crc=i:0 summary_offset_start=n:0 summary_start=n:116
